lib/hooks/useUserConnections.ts

```diff
@@ -3,24 +3,24 @@ import publicApiFetcher from "lib/utils/public-api-fetcher";
 import useSupabaseAuth from "./useSupabaseAuth";
 import { useToast } from "./useToast";
 
-interface UserCollaborationResponse {
-  data?: DbUserCollaboration[];
+interface UserConnectionResponse {
+  data?: DbUserConnection[];
   meta: Meta;
 }
 
-type collaborationRequestPayload = {
+type connectionsRequestPayload = {
   username: string;
   message: string;
 };
-const useUserCollaborations = () => {
+const useUserConnections = () => {
   const { sessionToken } = useSupabaseAuth();
   const { toast } = useToast();
-  const { data, error, mutate } = useSWR<UserCollaborationResponse, Error>(
+  const { data, error, mutate } = useSWR<UserConnectionResponse, Error>(
     "user/collaborations",
-    publicApiFetcher as Fetcher<UserCollaborationResponse, Error>
+    publicApiFetcher as Fetcher<UserConnectionResponse, Error>
   );
 
-  async function requestCollaboration(payload: collaborationRequestPayload) {
+  async function requestConnection(payload: connectionsRequestPayload) {
```

Comment:
Same here. The request sent to the API should remain the same

Reply:
The request are still the same here